src/modules/token/utils/index.ts

```diff
@@ -16,7 +15,7 @@ export const getTokenImage = ({
   if (isNativeToken) {
     return symbol === 'SDN' ? 'icons/sdn-token.png' : 'icons/astar.png';
   } else {
-    return getIcon({ icon: String(iconUrl), symbol });
+    return iconUrl || '';
```

Comment:
What it is going to be displayed on UI if method returns '' Is it going to be broken image? Maybe to put some default image

Reply:
I've modified in this [commit](https://github.com/AstarNetwork/astar-apps/pull/517/commits/e7316f100bf38892b26276584e9f5664d59db2d0).
<img width="1195" alt="image" src="https://user-images.githubusercontent.com/92044428/191460239-e22d9766-d74c-429f-a91f-9003a856f2f4.png">